### ISSUE TITLE
Data_id DEM has no time range

### ISSUE THREAD

**Issue by AliceBalfanz** [dcs4cop/xcube-sh#25]
When wanting receive data of data_id 'DEM' , the time_range is one of the required parameters:
![image](https://user-images.githubusercontent.com/23457217/101621445-0c995900-3a16-11eb-9584-9640fe810fb9.png)

However, when looking at the metadata of `time_range` it does not provide values, only `type` and `format` :
![image](https://user-images.githubusercontent.com/23457217/101621631-4c604080-3a16-11eb-817d-6068543b22e5.png)

Is there a way, that a user can find out, which time range to use in order to get the data?

**Comment by forman** [comment]
I'll consider this a bug. 

Fix: if dataset is `DEM`, then `time_range` should not be required.

**Comment by forman** [comment]
> Is there a way, that a user can find out, which time range to use in order to get the data?

No. The DEM is a composite, see https://docs.sentinel-hub.com/api/latest/data/dem/. 

In the output data "cube", we can simply assign the time range requested by the user as coordinates. This will avoid introducing entirely different handling of the "DEM" dataset:

    # Requested time range
    start, end = time_range
    center = start + (end - start) / 2
    # Time and time bounds coordinate
    if dataset_name == "DEM":
        time = xr.DataArray([center], dim=("time",), attrs=dict(bounds="time_bnds"))
        time_bnds = xr.DataArray([[start, end]], dim=("time", "bnds"))
        ...

**Comment by forman** [comment]
As of https://github.com/dcs4cop/xcube-sh/pull/74, DEM should be treated in a special way. E.g. return it without time coordinates.
